docs/index.md

```diff
@@ -129,7 +129,6 @@ After successfully installing Codacy on your cluster, you are now ready to perfo
 
 2.  Follow Codacy's onboarding process, which will guide you through the following steps:
 
-    -   Creating an administrator account
```

Comment:
The administrator account is automatically created during the first login. Maybe that information is still relevant.
Only the first user will be automatically admin.
Next admins have to be manually assigned by this person.

Note: This is platform admin, not confuse with organization admins.

Reply:
Thanks for the clarification @rtfpessoa.

Given this input I thinks it's best to still mention that this special Codacy admin account is created during the onboarding, but making it more explicit that it's done automatically.

I will make a proposal with this soon.